miner/worker_test.go

```diff
@@ -468,7 +470,9 @@ func testGetSealingWork(t *testing.T, chainConfig *params.ChainConfig, engine co
 	// This API should work even when the automatic sealing is enabled
 	w.start()
 	for _, c := range cases {
-		block, _, err := w.getSealingBlock(c.parent, timestamp, c.coinbase, c.random, nil, false)
+		// <specular modification>
+		block, _, err := w.getSealingBlock(c.parent, timestamp, c.coinbase, c.random, nil, false, nil, nil)
```

Comment:
might be misreading but is this setting a bool to nil (2nd to last param)

Reply:
i was actually passing an extra param. I did run `make test` to confirm all the tests pass. There must be a different way to build an run this test, I'll look into it.

Reply:
If we update to the latest geth, the new `getSealingBlock` api means we can eliminate the specular diff in this file.